README.md

```diff
@@ -699,7 +707,7 @@ import type { TestRunnerConfig } from '@storybook/test-runner';
 
 const config: TestRunnerConfig = {
   tags: {
-    include: [], // string array, e.g. ['test-only']
+    include: [], // string array, e.g. ['test']
```

Comment:
Might be worth pointing out here that this is the default.

Reply:
good one!